installing/installing_aws/installing-aws-network-customizations.adoc

```diff
@@ -60,6 +60,8 @@ include::modules/installation-aws-config-yaml.adoc[leveloffset=+2]
 include::modules/nw-modifying-operator-install-config.adoc[leveloffset=+1]
 include::modules/nw-operator-cr.adoc[leveloffset=+1]
 
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
```

Comment:
You have this file in different places in the AWS and Azure flows. I think I'd change this one.